### DISCUSSION TITLE
关于 Redis 最佳实践 中元数据的描述

### DISCUSSION THREAD

**Discussion by WellJay** [juicedata/juicefs#4909] [Q&A]
请问下关于redis存储元数据的说明【如果要存储 1 亿个文件，大约需要 30GiB 内存。】 
这里的 1 亿个文件，每个文件指的是的是JuiceFS文档中的 Chunk 64MB 还是 实际切割的 Block 4MB？

**Comment by williamstein** [comment]
I'm also very interested in how the redis size is a function of both the number of files and the number of chunks in those files.  E.g., 1 enormous file could have a million chunks, so presumably the amount of metadata stored must be a function of both the file sizes and the chunk size?
